pom.xml

```diff
@@ -4,7 +4,7 @@
     <groupId>org.alfresco.tas</groupId>
     <artifactId>restapi</artifactId>
     <name>alfresco-tas-restapi</name>
-    <version>1.29-SNAPSHOT</version>
+    <version>1.29</version>
```

Comment:
This will happen at the time of the release. Needs to be reverted to 1.29-SNAPSHOT.
Other than a minor comment, it looks good!